source/fundamentals/database-collection.txt

```diff
@@ -0,0 +1,334 @@
+
+The following code shows how to use the asynchronous
+``ListDatabaseNamesAsync()`` method or the synchronous ``ListDatabaseNames()`` method to
+list the names of the databases in a cluster:
+
+.. tabs::
+
+   .. tab:: Asynchronous
+      :tabid: list-db-async
+
+      .. code-block:: csharp
+         :copyable: true
+
+         await mongoClient.ListDatabaseNamesAsync();
+
+   .. tab:: Synchronous
+      :tabid: list-db-sync
+
+      .. code-block:: csharp
+         :copyable: true
+
+         mongoClient.ListDatabaseNames();
+
+Drop a Database
+---------------
+
+Dropping a database permanently deletes all the data in that database's
+collections. To drop a database, call the
+asynchronous `DropDatabaseAsync()
+<{+new-api-root+}/MongoDB.Driver/MongoDB.Driver.IMongoClient.DropDatabaseAsync.html>`__
+method or synchronous `DropDatabase()
+<{+new-api-root+}/MongoDB.Driver/MongoDB.Driver.IMongoClient.DropDatabase.html>`__
+method on your ``IMongoClient`` instance, passing the database name as
+the parameter.
+
+The following code shows how to use the asynchronous
+``DropDatabaseAsync()`` method or the synchronous ``DropDatabase()`` method to
+drop a database called ``test_db``:
+
+.. tabs::
+
+   .. tab:: Asynchronous
+      :tabid: drop-db-async
+
+      .. code-block:: csharp
+         :copyable: true
+
+         await mongoClient.DropDatabaseAsync("test_db");
+
+   .. tab:: Synchronous
+      :tabid: drop-db-sync
+
+      .. code-block:: csharp
+         :copyable: true
+         
+         mongoClient.DropDatabase("test_db");
+
+.. warning:: Dropping a Database Deletes Data
+
+   Dropping a database permanently deletes all
+   documents in the database's collections and all indexes on those collections.
+   After you drop a database, you cannot access or restore any of its data.
+
+Access a Collection
+-------------------
+
+You can access a collection by retrieving an `IMongoCollection
+<{+new-api-root+}/MongoDB.Driver/MongoDB.Driver.IMongoCollection-1.html>`__
+instance from your database. You can use an ``IMongoCollection``
+instance to perform data operations,
+create aggregations, and manage indexes. Call the `GetCollection()
+<{+new-api-root+}/MongoDB.Driver/MongoDB.Driver.IMongoDatabase.GetCollection.html>`__
+method on an ``IMongoDatabase`` instance to retrieve an ``IMongoCollection``.
+
+If you pass the name of a nonexistent collection to this method, the
+driver still returns an ``IMongoCollection`` instance. When you insert
+any data into this collection, the server creates it. To learn how to
+explicitly create a collection, see the :ref:`Create a Collection
+<csharp-create-collection>` section of this guide.
+
+This example uses the ``GetCollection()`` method to
+perform the following actions:
+
+- Access a collection called ``coll_xyz`` from a database referenced by
+  the ``myDB`` variable
+- Set a write concern on the collection in the ``MongoCollectionSettings`` type
```

Comment:
I'm not sure this example needs to set the write concern. It would be more clear and focused on accessing a collection if it only gets the "coll_xyz" collection.